modules/storage/images.yml

```
@@ -105,4 +105,33 @@ images:
     tag:
       - "24.06.0"
     destinations:
-      - registry.sighup.io/fury/netapp/trident-operator
\ No newline at end of file
+      - registry.sighup.io/fury/netapp/trident-operator
+
+  - name: Cloud Provider Vsphere - PV [Fury Vsphere Module]
+    source: registry.k8s.io/cloud-pv-vsphere/cloud-provider-vsphere
+    tag:
+      - "v1.29.0"
+      - "v1.30.0"
+      - "v1.30.1"
+      - "v1.31.0"
+      - "v1.31.1"
+    destinations:
+      - registry.sighup.io/fury/vsphere/cloud-provider-vsphere
```

let's follow our naming convention (`registry.sighup.io/fury/<upstream repo>/<upstream iamge>`):
```suggestion
      - registry.sighup.io/fury/cloud-pv-vsphere/cloud-provider-vsphere
```